artifacts/fake-templates/android-sdk.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: android-sdk
+  annotations:
+    openshift.io/display-name: Android SDK Installer!
+    description: "Installer to add Android SDK to the Build Farm (Mobile CI/CD). Before downloading the Android SDK Installer, you must agree to the terms and conditions of the Android Software Development Kit License Agreement, https://developer.android.com/studio/terms.html.\n\n"
+    iconClass: fa fa-android
+    tags: JENKINS,ci cd,mobile-service
+    template.openshift.io/long-description: Not sure if we could here _include_ the actual license, in a textbox/textarea.
+    template.openshift.io/provider-display-name: Feedhenry
+    template.openshift.io/documentation-url: https://github.com/aerogear/digger-jenkins
+    template.openshift.io/support-url: https://github.com/aerogear/digger-jenkins
+labels:
+  template: android-sdk
+objects:
+- apiVersion: v1
+  kind: Service
+  metadata:
+    name: android-sdk
+    labels:
+      app: android-sdk
+      template: android-sdk
+  spec:
+    ports:
+    - name: android-sdk
+      port: 3000
+      protocol: TCP
+      targetPort: 3000
+    selector:
+      run: android-sdk
+    sessionAffinity: None
+    type: ClusterIP
+parameters:
+  - name: LICENSE_AGREEMENT
+    description: You need to enter YES in order to continue.
+    from: "YES"
+    required: true
```

Comment:
@matzew @aidenkeating Can a `type` of `boolean` be used for the license accept? The UI looks like it should render it https://github.com/openshift/origin-web-catalog/blob/master/src/components/catalog-parameters/catalog-parameters.html

Reply:
@david-martin Yes a type can be set to accept the license. This is specified in the ticket to create the actual base APB. So this won't be a text field in reality.

Reply:
:+1: